.github/workflows/compose_test.yaml

```diff
@@ -89,26 +89,11 @@ jobs:
       && !cancelled()
       && fromJson(needs.changes.outputs.all).docs_all_modified_files
     steps:
-      - uses: actions/checkout@v4
-      - name: Set up Python
-        uses: actions/setup-python@v5
+      - uses: SciCatProject/scicatlive/.github/actions/mkdocs-pages@mike_action
```

Comment:
I will change this to main in the next PR, otherwise the workflow breaks in this one